README.md

```diff
@@ -55,6 +55,10 @@ GlobusClient.configure(
 def create_user_directory
   GlobusClient.mkdir(user_id: 'mjgiarlo@stanford.edu', path: 'mjgiarlo/work1234/version1')
 end
+
+def lookup_dir_contents
+  GlobusClient.get_filenames(path: '/uploads/mjgiarlo/work1234/version1')
```

Comment:
Also wondering about the need to have the explicit `/uploads/` here, which doesn't seem (⁉️) to be needed for other path operations, e.g., `Endpoint#mkdir`.

Reply:
I also wonder if we need a `user_id:` arg here like on line 56.

Reply:
Can you say more about what the `user_id:` arg would be needed for in this case? If the `get_filenames` request is coming from h2 so that it can create `attached_files`, it would be sending a user/work/version path it wanted to look up. This method would not need to care about the user_id.

Reply:
I believe it's a required argument when instantiating the `Endpoint` class behind the scenes, like what's on line 30 of `api_test.rb`. It won't be used for the ls operation, though, which might suggest a refactoring to tease apart access stuff from file transfer stuff, but I don't know that I'd want to bog your work down with that at this point.

Reply:
I agree it would be a good next step to separate the stuff that requires a user_id. As it is currently, h2 will need to send a `user_id` when getting a list of files in a particular version directory.